runtime/vm/visible.c

```diff
@@ -129,15 +129,19 @@ checkVisibility(J9VMThread *currentThread, J9Class* sourceClass, J9Class* destCl
 					 * host is loaded first and the class being accessed loads its
 					 * nest host after.
 					 */
-					if (NULL == sourceClass->nestHost) {
+					if ((NULL == sourceClass->nestHost)
```

Comment:
no changes needed here, the code currectly handles it by adding `J9_CURRENT_CLASS`